### ISSUE TITLE
Bench Training Max vs Close Grip Bench Training Max

### ISSUE THREAD

**Issue by acer589** [JonnyBeeGod/nSuns-5-3-1-iOS-issues#23]
It seems like these are just one variable, which is fine since I have no idea what my close bench max would be. But they’re labeled differently in app. It’s not really a problem once you figure it out, but I felt like an idiot for a minute trying to figure out why my bench press weight was messed up.

Easy possible solution is just change the label on the Close Grip Page to read “Bench Press Training Max”.

**Comment by JonnyBeeGod** [comment]
Good point, as long as these maxes are the same and not separate maxes for linked lifts like BP and CGBP they should be labeled both the same.

**Comment by JonnyBeeGod** [comment]
https://github.com/JonnyBeeGod/nSuns-5-3-1-iOS-issues/issues/16
